glide.yaml

```diff
@@ -65,4 +65,19 @@ import:
   version: 0fe204460da2c8fa1babcaac196e694de8f1aaa1
 
 - package: github.com/miekg/dns
-  version: db96a2b759cdef4f11a34506a42eb8d1290c598e
\ No newline at end of file
+  version: db96a2b759cdef4f11a34506a42eb8d1290c598e
+
+- package: github.com/shirou/gopsutil
+  version: 2.1.0
+  subpackages:
+  - load
+
+- package: github.com/davecgh/go-spew
+  version: 2df174808ee097f90d259e432cc04442cf60be21
+  subpackages:
+  - spew
+
+- package: golang.org/x/sys
```

Comment:
Why do we need to vendor `golang.org/x/sys`? It doesn't seem to be referenced anywhere in the code (neither within `github.com/shirou/gopsutil`)...